### ISSUE TITLE
Question - Will update brings bloatware back?

### ISSUE THREAD

**Issue by shumafuk** [Sycnex/Windows10Debloater#328]
It is not issue, just simple question. If I in the future install Windows updates on debloated system, will that brings bloat-ware back? Thanks.

**Comment by WormChickenWizard** [comment]
With normal security updates no. With feature updates perhaps. The last feature update I experienced didn't re-add any bloatware that had been already removed. They generally only add bloat that is respective to the new "features" of the feature update. (if you want to call it that)

**Comment by Gray-0men** [comment]
No, I wanted to make sure they couldn't reinstall after an update or a userprofile is added, idk why Microsoft wanted to do that and made it a sneaky re install but I added a bit to make sure that doesn't happen
